app/assets/stylesheets/all/components.scss

```diff
@@ -173,6 +174,17 @@ dl.order-summary {
   }
 }
 
+dl.descriptive-list-inline {
```

Comment:
Added semantic markup to a few descriptive lists. This styling lets us format the lists inline. Looks approximately like this, only with better spacing between the two items. (GH is converting even my non-breaking spaces to a single space)
**Pipeline** Pipeline name        **Other Key** Other value